.gitignore

```diff
@@ -14,7 +14,7 @@ nyc
 /coverage/tmp/
 
 # Temporary api-extractor build artifacts
-**/_api-extractor-temp/**
+**/_api-extractor-temp**/**
```

Comment:
Nit: If we saved the output to `_api-extractor-temp/v1`, `_api-extractor-temp/v2`, we wouldn't need to change this, and would result in fewer temp directories at the root.

Note that if we keep this change, we'll need to update the `clean` script in the package.json to clean up all of these new root directories.

Reply:
Sorry, I mistook this for the gitignore under `docs`. I understand now. We currently _download_ artifacts to `_api-extractor-tempv1, etc.`, rather than putting them under the same directory where we _generate_ API metadata when building the repo locally? This makes sense to me, at least for now. Sorry if I caused confusion.